web/themes/custom/vote_gov/templates/partial/registration-date-online.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * partial
+ * Voter online registration deadline
+ *
+ */
+#}
+
+{% if content.field_online_deadline is not empty %}
+    <li><span class="bold-text">{{ 'Online registration deadline: ' | t }}</span>{{ content.field_online_deadline | field_value }}</li>
```

Comment:
The online deadline moves to another line because that is how the Double Field module is styled. We need to style this div with inline:block, but I am not sure where it is best to do this since the styles are coming from a module.

Reply:
This is how I would handle this template. This uses the default filter to provide a fallback. Then you can check if there is a deadline before you print the content. Then it uses the striptags filter to remove extra markup coming from doublefield.
```
{% set deadline = content.field_online_deadline | field_value | default(content.field_g_online_deadline | field_value) %}

{% if deadline %}
  <li><span class="bold-text">{{ 'Online registration deadline: ' | t }}</span>{{ deadline | render | striptags }}</li>
{% endif %}
```